### ISSUE TITLE
tgp1.brazzersnetwork.com

### ISSUE THREAD

**Issue by Revivable2556** [hagezi/dns-blocklists#2369]
### Which AdBlocker/DNS cloud service do you use?

uBlock, NextDNS

### NextDNS users only

- [X] IMPORTANT - I have disabled the block page in NextDNS and confirm that the problem still occurs!

### With which block list(s) does the problem occur?

Multi PRO

### Which domain(s) should be unblocked?

tgp1.brazzersnetwork.com

### Why should the domain(s) be unblocked?

Blocking this domain causes brazzers.com to not load.

**Comment by hagezi** [comment]
Thank you @XRzero for your support. I can't reproduce it either. What exactly is not working @Revivable2556? More details please.

**Comment by Revivable2556** [comment]
Thank you @XRzero & @hagezi for looking into this. It appears that I am no longer having issues as of 3/19 when using Multi Pro w/Brazzers, so I'm not sure exactly what happened that led to the issues I ran into 3/17. I noticed that when I attempted to access the site that I was unable to (no page would load), while I noticed the domain above being blocked in my NextDNS logs. When I "downgraded" to using the Multi-Normal or AdGuard DNS filters - neither of which contains the domain above in their respective block lists - brazzers.com loaded without issue. 

I switched back to Multi-Pro and while I do not see the domain above in my logs anymore, I also have no issue accessing the site currently. I'm not fluent in filter creation and management, so I just assumed the issue I ran into on 3/17 was tied to the domain above? For additional information, the only DNS blocklist I had enabled was Multi-Pro, while accessing the site on a Windows 11 machine via Firefox w/uBlock Origin (w/AdGuard URL, Block LAN, and Easylist Annoyances activated).
